helm/cluster-cloud-director/templates/helpers/_workers.tpl

```diff
@@ -3,11 +3,13 @@ Generates template spec for worker machines.
 */}}
 {{- define "worker-vcdmachinetemplate-spec" -}}
 {{- $pool := $.nodePool.config | deepCopy -}}
+{{- $pool = set $pool "diskSize" ( include "calculateDiskBytes" $pool.diskSizeGB ) -}}
+{{- $pool = unset $pool "diskSizeGB" -}}
```

Comment:
this was missed in the previous cluster integration PR (to render worker resources)

Reply:
Where does that function come from?

Reply:
thanks for pointing out that i completely forgot to commit two files :facepalm: 

https://github.com/giantswarm/cluster-cloud-director/pull/374/files#diff-84ed6144d005d761e40cefbbcc342f5b12576a808d0cf6701bfe03cac8186233